examples/translation_example.py

```diff
@@ -0,0 +1,401 @@
+    ddf = cudf.DataFrame.from_pandas(ddf)
+    return ddf
+
+def combine_text_false(ddf):
+    ddf["translation"]=ddf["start_sym"] + ddf["indic_proc_text"] + ddf["end_sym"]
+    ddf["indic_proc_text"] = ddf["translation"]    
+    return ddf
+
+def atleast_letter(df):
+    df=df.to_pandas()
+    df['isalpha']=df['indic_proc_text'].apply(has_alphabet_characters)
+    df=cudf.DataFrame(df)
+    return df
+
+def post_output(output):
+    import torch.nn.functional as F
+    padded_tensor = output
+    if output.size(1) <  MAX_OUT_LEN:
+        pad_size = MAX_OUT_LEN - output.size(1)
+        padded_tensor  = F.pad(output, (0, pad_size), "constant", 1)
+    return padded_tensor
+
+def main_func(args):
+    st = time.time()
+    translation_config = TranslationConfig(
+        pretrained_model_name_or_path=args.pretrained_model_name_or_path,
+        max_length=256,
+        num_beams=5,
+        autocast=args.autocast,
+    )
+    global MAX_OUT_LEN
+    MAX_OUT_LEN = translation_config.max_length
+    rdst = time.time()
+    input_files = [
+        os.path.join(args.input_data_dir, x) for x in os.listdir(args.input_data_dir)
+    ]
+    
+    ddf = read_data(
+        input_files,
+        file_type=args.input_file_type,
+        backend="cudf",
+        files_per_partition=1,
+        add_filename=True,
+    )
+    
+    rdet = time.time()
+    ddf_meta = ddf._meta.copy()
+
+    ddf_meta['indic_proc_text'] = ""
+    ddf_meta['doc_id'] = ""
+    ddf_meta['start_sym'] = ""
+    ddf_meta['end_sym'] = ""
+    ddf_meta = ddf_meta.drop('text',axis=1)
+    ddf = ddf.map_partitions(process_input_text, meta=ddf_meta)
+    ddf['word_count'] = ddf['indic_proc_text'].str.split().list.len()
+    ddf['word_count'] = ddf['word_count'].astype('int64')
+
+    ddf_true = ddf[(ddf['word_count'] <= translation_config.max_words_per_sen)]
+    ddf_metaa = ddf_true._meta.copy()
+    ddf_metaa['isalpha']=False
+    has_letter = ddf_true.map_partitions(atleast_letter, meta=ddf_metaa)
+
+    ddf_trans = ddf_true[has_letter['isalpha']]
+    ddf_false = ddf_true[~has_letter['isalpha']]
+
+    ddf=ddf_trans.drop(columns='word_count')
+
+    ## ddf false operations
+    ddf_false=ddf_false.drop(columns='word_count')
+    ddf_false_meta = ddf_false._meta.copy()
+    ddf_false_meta["translation"]=""
+    ddf_false = ddf_false.map_partitions(combine_text_false, meta=ddf_false_meta)
+
+    ddf['indic_proc_text'] = ddf['indic_proc_text'].astype('str')
+    ddf = ddf.map_partitions(preprocess_df, meta=ddf_meta)
+
+    columns = ddf.columns.tolist()
+    model = ModelForSeq2SeqModel(translation_config)
+
+    pipe = op.Sequential(
+        op.Tokenizer(model, cols=[args.input_column], tokenizer_type="sentencepiece"),
+        op.Predictor(
+            model,
+            sorted_data_loader=True,
+            batch_size=args.batch_size,
+            pred_output_col="translation",
+            post=post_output,
+        ),
+        keep_cols=columns,
+    )
```

Comment:
We can make this follow this example, like this model is a sequence to sequence model under the hood.  

https://github.com/NVIDIA/NeMo-Curator/blob/main/nemo_curator/classifiers/aegis.py